{{cookiecutter.project_name}}/docs/_pages/template/linters.rst

```diff
@@ -79,4 +79,5 @@ yamllint
 ~~~~~~~~
 
 This linter is used to lint your `yaml` files.
+Just run ``yamllint .``.
```

Comment:
It is a good practice (I try really hard to follow it) not to start your sentences with "just".